en/0-intro.md

```diff
@@ -2,38 +2,33 @@
 
 ## Abstract
 
-Today’s work market lacks efficiency because of high transaction costs[^1][^2][^3] and frequent conflicts of interests between the employer and the employee.
+The current work market is inefficient due to high transaction costs and frequent conflicts of interest between employers and employees. 
 
-The WoM (World of Meeds) proposes a new paradigm for reducing transaction costs and reinventing working relationships. It bridges the gap between traditional companies and new forms of organisation by proposing a fair and transparent incentive system that realigns the interests of all parties.
+The World of Meeds (WoM) proposes a new paradigm to address these issues by reducing transaction costs and reinventing working relationships. It's a digital platform that bridges the gap between traditional companies and new forms of organization by introducing a fair and transparent incentive system that realigns the interests of all parties.
 
 ## Introduction
 
-The work market today is broken. It fails to satisfy both workers and employers.
```

Comment:
I feel it is more appropriate to replace "broken" with the term **dysfunctional.** BROKEN is ambiguous in the sense that it could equally mean disintegration of the work market, which does not perfectly portray the current failing state of the work market.

Reply:
@Amby01 no need to comment on red sentences. The red sentences are those that are being replaced by the green ones. 
Hence, the proposed change has already got rid of the "broken" term.